### ISSUE TITLE
The feature extraction technique (or the input to the neural network model) used in this implementation?

### ISSUE THREAD

**Issue by Xinghui-Wu** [speechbrain/speechbrain#857]
The papers with regard to DeepSpeech 1 and 2 from Baidu claimed that they used the spectrogram of power normalized audio clips as the features to the system.
The implementation of Mozilla DeepSpeech, based on TensorFlow, said that they turned to apply the MFCC features.
Now I am curious about that of your implementation for my research purpose on the comparison of various feature extraction algorithms. So, I will appreciate it if you could solve my puzzle. Thank you.

**Comment by mravanelli** [comment]
Hi @Xinghui-Wu, SpeechBrain supports a variety of features (e.g, spectrograms, FBANKs, MFCCs). The one used the most are FBANKs, whose computations are reported here: https://github.com/speechbrain/speechbrain/blob/develop/speechbrain/lobes/features.py#L18

**Comment by Xinghui-Wu** [comment]
> Hi @Xinghui-Wu, SpeechBrain supports a variety of features (e.g, spectrograms, FBANKs, MFCCs). The one used the most are FBANKs, whose computations are reported here: https://github.com/speechbrain/speechbrain/blob/develop/speechbrain/lobes/features.py#L18

Thank you a lot!
I have one more question please. I have noticed that you have published a few pre-trained models on the website, https://huggingface.co/speechbrain, which seem to be trained on the FBank feature according to the hyperparams.yaml file. So I wonder if there are any other published pre-trained models that use MFCC feature?

**Comment by mravanelli** [comment]
For speech recognition we normally use FBANKs. MFCCs are not used that much anymore because they often do not lead to performance improvements when feeding them into a neural network. In other contexts such as speech enhancement,  we use spectrogram directly. In other cases (e.g, speech separation) we use the raw samples without any feature extraction step.
